src/update-path.js

```diff
@@ -15,12 +15,33 @@ const updateAbsolutePath = async (hostSite, filePath, outputPath) => {
       for (var i = 0; i < length; i++) {
         const { name, value } = node.attributes[i];
         // do stuff...
+        var urlPattern = /^(?:\/|[a-z]+:\/\/)/;
```

Comment:
Do you have any article about this: `var urlPattern = /^(?:\/|[a-z]+:\/\/)/;` ?

Also, `urlPattern` start with `/` and not start with `//` is relative path.

Reply:
tui coi trên solution này: https://stackoverflow.com/a/17819167
cũng có tìm ra được một solution nữa: https://stackoverflow.com/a/10687158, mà nó dính chữ "i" sợ bị đụng thằng i trong vòng lặp for, với không biết chữ "i" đó dùng làm gì nên không xài.

Reply:
OK